### PR TITLE
contrib/thinkfan: new package (1.3.1) ← dep: contrib/yaml-cpp: new package (0.8.0)

### PR DESCRIPTION
**Pull request by valpackett**
My first packages :wave: 

https://github.com/vmatare/thinkfan and its dependency https://github.com/jbeder/yaml-cpp

### CODE REVIEW

File: contrib/thinkfan/files/thinkfan [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# thinkfan service
+
+type            = bgprocess
```
Comment (q66) [inline contrib/thinkfan/files/thinkfan:3]: can we make this a process service instead?

File: contrib/thinkfan/template.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
+pkgname = "thinkfan"
+pkgver = "1.3.1"
+pkgrel = 0
+build_style = "cmake"
+hostmakedepends = ["cmake", "ninja", "pkgconf"]
+makedepends = ["yaml-cpp-devel"]
+pkgdesc = "Simple, lightweight fan control program"
```
Comment (q66) [inline contrib/thinkfan/template.py:7]: just "Lightweight fan control program" should be descriptive enough

File: contrib/yaml-cpp/template.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
+pkgname = "yaml-cpp"
+pkgver = "0.8.0"
+pkgrel = 0
+build_style = "cmake"
+configure_args = ["-DYAML_BUILD_SHARED_LIBS=ON"]
+hostmakedepends = ["cmake", "ninja", "pkgconf"]
+pkgdesc = "YAML parser and emitter in C++ matching the YAML 1.2 spec"
```
Comment (q66) [inline contrib/yaml-cpp/template.py:7]: i'd make this terser, like "C++ YAML parser and emitter"

**Comment by q66** [comment]
outside of these, please fix the lint errors

File: contrib/thinkfan/files/thinkfan (hunk 1 above)
Comment (valpackett) [reply contrib/thinkfan/files/thinkfan:3]: kinda, it wouldn't write to syslog in that case but I guess that's fine

Comment (q66) [reply contrib/thinkfan/files/thinkfan:3]: can always just log to a file

**Comment by valpackett** [comment]
done (just silenced the logging as it's not useful)

**Comment by q66** [comment]
lint fails

**Pull request merged**